src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -8,34 +8,17 @@
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
     <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
-    <!-- Avoid unused fields warnings in Unix build -->
-    <NoWarn Condition="'$(TargetPlatformIdentifier)' != 'windows'">$(NoWarn);CA1823</NoWarn>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' != 'windows'">SR.PlatformNotSupported_Registry</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Do we know why the `$(NetCoreAppCurrent)` target framework wasn't a PNSE throwing assembly? Looks to me that this assembly actually worked on Unix before (even though it doesn't make sense, because of the Win32 dependency).

Reply:
cc @ericstj

Reply:
> Looks to me that this assembly actually worked on Unix before

Not really:
https://github.com/dotnet/runtime/pull/72667#discussion_r974375107